src/app/chip/chip.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-content></ng-content>
+<span class="close" (click)="onDelete()" *ngIf="close">
+  <svg fill="#000000" height="32" viewBox="0 0 24 24" width="16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe we should use the close icon from Material Icons instead of svg?

Reply:
My point was to not rely on the library used (mdi vs material icon) but i guess it is in vain if the chip input use the material icon ... i could change it back